File: Server/app/model/metrics.py
class Metrics:
    accuracy = 0
    precision = 0
    recall = 0
    f1 = 0
    rmse = 0
    mae = 0
    silhouette = 0

    def __init__(self, accuracy, precision, recall, f1, rmse, mae, silhouette):
        self.accuracy = accuracy
        self.precision = precision
        self.recall = recall
        self.f1 = f1
        self.rmse = rmse
        self.mae = mae
        self.silhouette = silhouette

File: Client/src/app/model/metrics.ts
export class Metrics {
    accuracy: number;
    precision: number;
    recall: number;
    f1: number;
    rmse: number;
    mae: number;
    silhouette: number;

    constructor(){
        this.accuracy = 0;
        this.precision = 0;
        this.recall = 0;
        this.f1 = 0;
        this.rmse = 0;
        this.mae = 0;
        this.silhouette = 0
    }
}